eval.go

```diff
@@ -202,9 +187,14 @@ func (e *CallExpr) eval(app *App, args []string) {
 			}
```

Comment:
Yeah, sounds good. It's out of scope here, though, so I can either fix it later or leave it to you.